packages/table/preview/index.tsx

```diff
@@ -45,7 +45,15 @@ ReactDOM.render(<Nav selected="index" />, document.getElementById("nav"));
 
 function getTableComponent(numCols: number, numRows: number, columnProps?: any, tableProps?: any) {
     // combine table overrides
-    const tablePropsWithDefaults = { numRows, ...tableProps };
+    const getCellData = (row: number, col: number) => {
+        return Utils.toBase26Alpha(col) + (row + 1);
+    };
+
+    const onCopy = () => {
+        console.log("did copy");
```

Comment:
Oh wait I can't do this because it's not passing that info in -- it just gets success. I'm not sure what I'd pass to it actually. Lemme think about it.